### PR TITLE
Update .wordlist.txt

### PR DESCRIPTION
**Pull request by Rmalavally**
Updated word list

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -103,3 +103,9 @@ DGEMM
 HPCG
 HPL
 IOPM
+amdclang
+asan
+complilation
```
Comment (saadrahim) [inline .wordlist.txt:108]: compilation, this line shouldn't be added

Comment (Rmalavally) [reply .wordlist.txt:108]: Agreed. Please remove 'compilation'.

Comment (saadrahim) [reply .wordlist.txt:108]: You need to remove the line.

**Review by samjwu — CHANGES_REQUESTED** [review]
Now that https://github.com/RadeonOpenCompute/ROCm/pull/2338 is passing, I don't think we should add these to the word list.

We want these keywords to be wrapped in tilde.

File: .wordlist.txt
Comment (samjwu) [inline .wordlist.txt:1]: These words were being flagged intermittently:

uninstrumented
detections

**Comment by samjwu** [comment]
Fixed/added in https://github.com/RadeonOpenCompute/ROCm/pull/2338

**Pull request closed without merging**